### DISCUSSION TITLE
内容抄袭举报

### DISCUSSION THREAD

**Discussion by XdpCs** [AmazingAng/WTF-Solidity#824] [General]
欢迎提供抄袭链接，社区的朋友们一起举报

**Comment by JesseAtSZ** [comment]
solidity101课程第十四章，在CSDN发现了复制的文章：
https://blog.csdn.net/djklsajdklsajdlk/article/details/140382619
https://blog.csdn.net/qq_52708261/article/details/127026257
https://blog.csdn.net/qq_33583069/article/details/129751004
